packages/-ember-data/tests/integration/adapter/rest-adapter-test.js

```diff
@@ -313,8 +313,14 @@ module('integration/adapter/rest_adapter - REST Adapter', function (hooks) {
   });
 
   test('updateRecord - hasMany relationships faithfully reflect simultaneous adds and removes', async function (assert) {
-    Post.reopen({ comments: DS.hasMany('comment', { async: false }) });
+    Post = DS.Model.extend({
```

Comment:
we should scope the variable to the test now that the definition is entirely within it.

```suggestion
    const Post = Model.extend({
```

Also, since this test file already uses the newer non `DS.` way of accessing the `Model` class and the `attr` `belongsTo` and `hasMany` functions we should use those anywhere we are adding a new definition. This can be done in a later pass if you want.